templates/Web/Pages/Vue.MasterDetail/src/components/MasterDetailSideBarTab.vue

```diff
@@ -1,10 +1,10 @@
 ﻿<template>
   <button
-    @click="$emit('onDisplayTabClick', index)"
-    type="button"
     class="list-group-item list-group-item-action styles.sidebarText"
+    type="button"
+    @click="$emit('onDisplayTabClick', index)"
   >
-    <img src="@/assets/GreyAvatar.svg" alt="Default Grey Avatar" class="mr-3" />
+    <img class="mr-3" src="@/assets/GreyAvatar.svg" alt="Default Grey Avatar" />
```

Comment:
You move the class in Master Detail component, then you should have moved the class in Grid component too

Reply:
I will be fixing it in the next VueJS PR